qa/workunits/objectstore/test_fuse.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -ex
+#!/bin/bash -ex
```

Comment:
we removed the last bit of bashim in 15ea140b3bbf6b1bfa9eea19b0f167141fcaa2c9, why would we want to switch to bash again?